tacs/constitutive.pyx

```diff
@@ -2007,6 +2025,37 @@ cdef class LamParamShellConstitutive(ShellConstitutive):
         self.ptr = self.cptr
         self.ptr.incref()
 
+cdef class LamParamAllShellConstitutive(ShellConstitutive):
+    cdef TACSLamParamAllShellConstitutive* lam_cptr
+    def __cinit__(
+            self,
```

Comment:
Could you add a docstring here following the format of the other cython classes/methods